course_info/icommons.py

```diff
@@ -92,6 +108,31 @@ def _get_schools(self, school_id=None, **kwargs):
             raise ICommonsApiValidationError(str(e))
         return rv
 
+    def _course_info_instructor_list(self, course_instance_id=None, **kwargs):
+
+        # use kwargs to append 'people' to the resource url and to get course people data
+        kwargs['append_to_path'] = 'people'
+
+        instructors = None
+        rv = self._get_resource('course_instances', course_instance_id, **kwargs)
+        try:
+            if 'results' in rv:
+                instructors = []
+                for instance in rv['results']:
+                    course_person_schema(instance)
+                    if instance.get('role')['role_id'] in INSTRUCTOR_ROLE_IDS:
```

Comment:
If `role` isn't found, this will throw an exception finding `role_id`, which will raise an uncaught `RuntimeException` -- are we sure that's ok or will it result in a 500 server error?